modules/migrations/base/pullrequest.go

```diff
@@ -31,7 +31,6 @@ type PullRequest struct {
 	MergeCommitSHA string
 	Head           PullRequestBranch
 	Base           PullRequestBranch
-	Assignee       string
 	Assignees      []string
```

Comment:
So both `issue` and `pullrequest` have `Assignees` field?

Reply:
yes

Reply:
and since this tructs are only used for the migration functions internaly we dont have to worry about strange cases, this the downloaders should handle

Reply:
PS: Assignees are not handled at all at the moment